tests/components/axis/test_light.py

```diff
@@ -72,8 +80,10 @@ async def test_lights(hass):
 
     assert len(hass.states.async_entity_ids(LIGHT_DOMAIN)) == 1
 
-    light_0 = hass.states.get(f"light.{NAME}_ir_light_0")
-    assert light_0.state == "on"
+    entity_id = f"{LIGHT_DOMAIN}.{NAME}_ir_light_0"
+
+    light_0 = hass.states.get(entity_id)
+    assert light_0.state == STATE_ON
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)